apps/frontend/src/pages/tap.tsx

```diff
@@ -63,15 +63,6 @@ const TapPage: React.FC = () => {
               return;
             }
 
-            // If user is not logged in, direct them to sign in
```

Comment:
If we're not doing location taps in accountless then why did we remove this?

Reply:
Adding back in.

Reply:
added back, tested and it works, addressed